nimbus-ui/nimbusui/src/index.html

```diff
@@ -18,6 +18,7 @@
     <script src="node_modules/zone.js/dist/zone.js"></script>
     <script src="node_modules/reflect-metadata/Reflect.js"></script>
     <script src="node_modules/systemjs/dist/system.src.js"></script> -->
+    <script src="clientExtension.js"></script>
```

Comment:
Change the name to app_scripts.js to be consistent with styles file (app_styles.css)

Reply:
@dinakarmeda  - seems like a change that came in when merged with 1.1.0.M2. 
@vivekkamineni2018, @purnamashetty   - could you please check this, as it could be related to one of your PR's